stock_prices/stock_prices.py

```diff
@@ -3,7 +3,21 @@
 import argparse
 
```

Comment:
Great progress! The only test I noticed this solution doesn't pass is when given [100, 90, 80, 50, 20, 10] as input. It expects -10 but instead receives 0 according to the test.